boards/sodaq-one/board.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017 Kees Bakker, SODAQ
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sodaq-one
+ * @{
+ *
+ * @file
+ * @brief       Board common implementations for the SODAQ ONE board
+ *
+ * @author      Kees Bakker <kees@sodaq.com>
+ * @}
+ */
+
+#include "cpu.h"
+#include "board.h"
+#include "periph/gpio.h"
+
+void board_init(void)
+{
+    /* initialize the on-board LEDs, switch them off to start with */
+    LED_GREEN_OFF;
```

Comment:
I don't understand why you need to call this macro before initializing the pin

Reply:
Well, the reason is to have a defined (low) value before enabling it as output. This trick is mostly used to avoid a spike when enabling some output port. For a LED it doesn't really matter.